openstack-manager/README.md

```diff
@@ -81,20 +81,29 @@ Create a virtual machine for installing the Cloudify Manager on.
 
 * Make sure that you connect the VM to the `cloudify-management` security group.
 
-### Step 2: Bootstrap
+### Step 2: RPM installation
 
-Use the instructions provided in the [Manager Bootstrapping lab](../bootstrap) to perform the bootstrap.
+Use the instructions provided in the [Manager Installation  lab](../manager-installation) to perform the bootstrap.
 
 ### Step 3: Upload plugins
  
 Once bootstrapping is complete, upload the OpenStack plugin package:
 
+CLI
 ```bash
 cd ~
-curl -J -O https://github.com/cloudify-cosmo/cloudify-openstack-plugin/releases/download/2.2.0/cloudify_openstack_plugin-2.2.0-py27-none-linux_x86_64-centos-Core.wgn
-cfy plugins upload ~/cloudify_openstack_plugin-2.2.0-py27-none-linux_x86_64-centos-Core.wgn
+curl -L -J -O https://github.com/cloudify-cosmo/cloudify-openstack-plugin/releases/download/2.6.0/cloudify_openstack_plugin-2.6.0-py27-none-linux_x86_64-centos-Core.wgn
+curl -L -J -O http://www.getcloudify.org/spec/openstack-plugin/2.6.0/plugin.yaml
+cfy plugins upload ~/cloudify_openstack_plugin-2.6.0-py27-none-linux_x86_64-centos-Core.wgn -y plugin.yaml
```

Comment:
I think it'd be best to use the URL's directly.
Also, I think the latest version is 2.7.0 or 2.7.1 or something.

`cfy plugins upload http://repository.cloudifysource.org/cloudify/wagons/cloudify-openstack-plugin/2.7.1/cloudify_openstack_plugin-2.7.1-py27-none-linux_x86_64-centos-Core.wgn -y http://www.getcloudify.org/spec/openstack-plugin/2.7.1/plugin.yaml`